cdap-integration-test/src/test/java/co/cask/cdap/test/IntegrationTestBaseTest.java

```diff
@@ -40,4 +49,26 @@ public void testFlowManager() throws Exception {
     flowManager.stop();
   }
 
+  @Test
+  public void testDeployApplicationInNamespace() throws Exception {
+    Id.Namespace namespace = new Id.Namespace ("Test1");
+    createNamespace(new NamespaceMeta.Builder().setName(namespace).build());
```

Comment:
That's so you can pass other fields like ``description`` or ``NamespaceConfig`` for the namespace you are creating. Perhaps we can also add an overloaded field that only takes ``Id.Namespace`` and uses defaults for ``description`` and ``NamespaceConfig``.